02-express-tutorial/app.js

```diff
@@ -1 +1,60 @@
-console.log('Express Tutorial')
+const express = require('express');
+
+const app = express();
+
+app.use(express.static("./public"));
+
+app.get('/api/v1/test', (req, res) => {
+    res.json({ message: "it worked!"});
+});
+
+//Route to get a product by ID
+app.get('/api/v1/products/:productID', (req, res) => {
+    const { productID } = req.params;
+    const idToFind =parseInt(req.params.productID, 10);
+   
+    const product = products.find((p) => p.id === idToFind);
```

Comment:
Where is `products` coming from? It looks like a `require` might be missing.

Reply:
I have included 'require'.